standards/migrations/0007_auto_20191021_1519.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('standards', '0006_auto_20170929_1528'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='standard',
+            name='gradeband',
+            field=models.ForeignKey(related_name='standards', to='standards.GradeBand'),
```

Comment:
this change appears to be a side effect of running `./manage.py makemigrations` without any of the changes listed in this PR. I'm including it here since it seems like otherwise it'll eventually get included in whatever other PR tries to make migrations next.